docs/notes/2025.11.10.11.31.10.md

```
@@ -0,0 +1,113 @@
+
+  A[Agents (Thinker/Prover/…)]:::agent
+  R[Profiles (daimoi / nooi)]:::prof
+  T[Tasks (mem:*)]:::task
+  Sc[Schemas (MemoryEvent, Manifest, Snapshot)]:::schema
+
+  S1 --> S2 --> P1 --> I1
+  P1 --> I2
+  P1 --> I3
+  A -->|Query, Context| C --> P2
+  R -->|weights/budgets| P2
+  P2 -->|serialize| S2
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)